### DISCUSSION TITLE
Whisper cloud very slow recently

### DISCUSSION THREAD

**Discussion by osamabinsaleem** [deepgram/community#1005] [General help]
We're noticing that whisper cloud is performing very poorly in terms of speed for the past couple of weeks. What changed on Deepgram's end that's cauing this? We use language detection and have a requirement of many languages and that's why we can't use Nova2. This is a job id of a call recordinfg 555a2786-6181-4d71-a597-32467c1b11ec and it took around 20 minutes to transcribe. This is just too much

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).

**Reply by osamabinsaleem** [reply]
working with python
